pom.xml

```diff
@@ -7,24 +7,30 @@
     <version>4.2</version>
     <relativePath />
   </parent>
+
   <groupId>io.jenkins.plugins</groupId>
   <artifactId>theme-manager</artifactId>
   <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
+
   <properties>
     <revision>0.6</revision>
     <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.222.1</jenkins.version>
+    <bom.artifactId>2.222.x</bom.artifactId>
```

Comment:
Why put these here they are never used any where else?

Reply:
Hi timja,
no particular reason realy, kind of copied the format from the plugin template (and what is found within jenkinsci/bom/sample-plugin).
Want me to revert it?

Reply:
Yes please

Reply:
Done